### DISCUSSION TITLE
Funkmodul

### DISCUSSION THREAD

**Discussion by hsuliman2** [tbnobody/OpenDTU#1761] [Q&A]
Hi zusammen, seit einer Woche bekomme ich eine Fehlermeldung: "Es konnte keine Verbindung zu einem der konfigurierten Funkmodule hergestellt werden. Bitte überprüfen Sie die Verdrahtung."
weit einer wieso ? ichj habe sogar ein neues Funkmodul angeschlossen, aber das gleiche Problem.
Version: 24.2.16
Funkmodul: NRF24+

**Comment by D3R-ST3FAN** [comment]
Die PINs des NRF passen zum Pin-Mapping?

**Comment by hsuliman2** [comment]
Ja, es wurde an den PINs seit einem Jahr nichts geändert. Ich habe die geprüft.

**Comment by D3R-ST3FAN** [comment]
Ich meinte die Mappings im Profil unter Hardware, prüf das doch noch mal, ob das dazu passt wie das Funkmodul angeschlossen ist.
Evtl.. ist ein falsches Profil gewählt.

**Comment by hsuliman2** [comment]
Standardprofil , aber ich weiß nicht wie ich prüfen soll. Die Anschlüsse an Hardware wie in den Dokus.
<img width="800" alt="image" src="https://github.com/tbnobody/OpenDTU/assets/122373793/e5be4974-4433-4470-bb51-3ee087849768">

**Comment by MetaChuh** [comment]
hi @hsuliman2 

alles was in `Einstellungen > Hardware` **rosa** hinterlegt ist, weißt auf ein **pin_mapping.json problem** hin, **bzw**. dass die daten der config **noch nicht übernommen** wurden.

soll screenshot als beispiel:
wenn deine config passt, wirst du in **zwei spalten**, die soll und ist gpio ports sehen ...
**erst wenn links und rechts** (Ausgewählt und Aktiv) **beide settings identisch sind**, passen die einstellungen.
(sofern die verdrahtung mit der einstellung übereinstimmt)

(side info: die gewählten gpio's in diesem screenshot, sind spezifisch für ESP32 D1 Mini D0WD-V3, und entsprechen nicht dem standard)

<img width="1241" alt="Bildschirmfoto 2024-02-19 um 18 37 05" src="https://github.com/tbnobody/OpenDTU/assets/45773305/497b4cbb-626c-4f29-b1cd-aec0b48ec369">

hope it helps
greetings
